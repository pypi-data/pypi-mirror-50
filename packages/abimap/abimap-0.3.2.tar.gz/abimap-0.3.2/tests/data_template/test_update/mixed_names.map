# Map with releases with mixed names

NAME_1_0_0
{
    global:
        one_symbol;
    local:
        *;
} ;

ROGUE_RELEASE_4_5_6
{
    global:
        three_symbol;
} ;

OTHER_DIFFERENT_NAME_1_1_0
{
    global:
        two_symbol;
} NAME_1_0_0;
