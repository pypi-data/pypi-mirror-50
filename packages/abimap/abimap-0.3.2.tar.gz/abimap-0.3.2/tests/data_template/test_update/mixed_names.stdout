Added:
    symbol

# This map file was updated with PROGRAM_NAME_VERSION

ROGUE_RELEASE_4_5_6
{
    global:
        three_symbol;
} ;

ROGUE_RELEASE_4_6_0
{
    global:
        symbol;
} ROGUE_RELEASE_4_5_6;

NAME_1_0_0
{
    global:
        one_symbol;
    local:
        *;
} ;

OTHER_DIFFERENT_NAME_1_1_0
{
    global:
        two_symbol;
} NAME_1_0_0;

